Assets/HoloToolkit/Input/Scripts/ControllerVisualizer.cs

```diff
@@ -311,7 +312,7 @@ public GameObject SpawnTouchpadVisualizer(Transform parentTransform)
             {
                 touchVisualizer = GameObject.CreatePrimitive(PrimitiveType.Sphere);
                 touchVisualizer.transform.localScale = new Vector3(0.0025f, 0.0025f, 0.0025f);
-                touchVisualizer.GetComponent<Renderer>().material.shader = GLTFShader;
+                touchVisualizer.GetComponent<Renderer>().material = GLTFMaterial;
```

Comment:
the touch visualizer needs to be able to clean up the material we've 'created' here for when the controller gets destroyed. Otherwise I suspect we have a small leak when turning controllers off and on.

Reply:
Yeah we should be using `SharedMaterial` not `Material` call here

Reply:
My hesitation with using `SharedMaterial` is that I'd then be changing the material for every object that uses the default material. I instead added a Destroy call in ControllerInfo's destroy, to clean up after itself, but let me know if you prefer `SharedMaterial` instead.

Reply:
I do prefer `SharedMaterial` but as long as we're cleaning up after ourselves it should be okay.